maps/HDRUK/3.0.0/HDRUK/datasetv2/translation.jsonata

```diff
@@ -85,7 +85,7 @@ $map_deliveryLeadTime := {
   "2-4 WEEKS"       : "2-4 weeks",
   "1-2 MONTHS"      : "1-2 months",
   "2-6 MONTHS"      : "2-6 months",
-  "MORE 6 MONTHS"   : "More than 6 months",
+  "MORE 6 MONTHS"   : "6 months plus",
```

Comment:
I'm not sure about this.  The enum in the schema has `More than 6 months`, as does Chris' mapping file.

Reply:
It was giving me an error when pointed at the dev schema. Will have to dig